set.mm

```diff
@@ -26678,8 +26678,8 @@ choice between (what we call) a "definitional form" where the shorter
     $( Formula-building lemma for use with the Distinctor Reduction Theorem.
        (Contributed by Mario Carneiro, 8-Oct-2016.) $)
     drnfc2 $p |- ( A. x x = y -> ( F/_ z A <-> F/_ z B ) ) $=
-      ( vw weq wal cv wcel wnf wnfc eleq2d drnf2 albidv df-nfc 3bitr4g ) ABHAIZ
-      GJZDKZCLZGITEKZCLZGICDMCEMSUBUDGUAUCABCSDETFNOPCGDQCGEQR $.
+      ( vw weq wal cv wcel wnf wnfc eleq2d drnf2 dral2 df-nfc 3bitr4g ) ABHAIZG
+      JZDKZCLZGITEKZCLZGICDMCEMUBUDABGUAUCABCSDETFNOPCGDQCGEQR $.
```

Comment:
Keep the old proof here. It is actually strange that "minimize_with" produced this result... Anyone understands why ?

Reply:
Answer:
```
Proof of "drnfc2" stayed at 129 bytes using "dral2".
    (Uncompressed steps decreased from 78 to 75).
```
I would still keep albidv, so maybe add a proof discouragement tag ?

Reply:
Here the minimizer is referring to the total amount of bytes (labels inlcuded), but maybe for minimizations we are interested in the size of the upper-case section only. I'm not sure whether there was an agreement in the past about which counting matters most.

Reply:
Yes, in case of tie, it's the block of capital letters. Since this may go against the action of `MINIMIZE_WITH` as in the present case, this is not very satisfying and the situation could be improved. But here, using albidv is definitely more "elementary" and natural than using ~dral2.